test/test_nn.py

```diff
@@ -7867,7 +7867,7 @@ def _gelu_ref(X):
                 res = F.gelu(X)
                 ref = _gelu_ref(X.to(numpy_dtype).cpu().detach().numpy())
                 self.assertEqual(res, ref, rtol=rtol, atol=atol)
-                if dtype != torch.bfloat16:
+                if dtype == torch.float64:
```

Comment:
What is this change?

Reply:
_test_gelu is only being called with bfloat16, float32, and float64. This change just makes it so that we won't test float32 for gradcheck anymore